ur_robot_driver/CMakeLists.txt

```
@@ -104,36 +116,56 @@ add_executable(robot_state_helper
   src/robot_state_helper_node.cpp
   src/urcl_log_handler.cpp
 )
-target_link_libraries(robot_state_helper ur_client_library::urcl)
-ament_target_dependencies(robot_state_helper ${${PROJECT_NAME}_EXPORTED_TARGETS} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+target_link_libraries(robot_state_helper PUBLIC
+  ${std_msgs_TARGETS}
+  ${std_srvs_TARGETS}
+  ${ur_dashboard_msgs_TARGETS}
+  rclcpp::rclcpp
+  rclcpp_action::rclcpp_action
+  ur_client_library::urcl
+)
 
 add_executable(urscript_interface
   src/urscript_interface.cpp
 )
-ament_target_dependencies(urscript_interface ${${PROJECT_NAME}_EXPORTED_TARGETS} ${THIS_PACKAGE_INCLUDE_DEPENDS})
+target_link_libraries(urscript_interface
```

```suggestion
target_link_libraries(urscript_interface PUBLIC
```
Should this have a PUBLIC tag as well.